### ISSUE TITLE
VersionPress console not working with HTTPS redirect

### ISSUE THREAD

**Issue by Elethier** [versionpress/support#118]
I got VersionPress up and running a while ago, but I recently switched my site over to use HTTPS and now I'm running into some issues. I adjusted my server settings to force web browsers to redirect to HTTPS instead of serving content over HTTP. But now the VersionPress tab in the admin console is not working. Looking in the Chrome web inspector, I'm seeing a few errors along the lines of Failed to load: Response for preflight is invalid (redirect). Is this an issue I can easily fix with some configuration changes to VersionPress or am I going to have to turn off the HTTPS redirect?

- VersionPress version: 3.0.2
- WordPress version: 4.8.2
- OS Version: Linux ip-172-26-11-148 3.13.0-100-generic #147-Ubuntu SMP Tue Oct 18 16:48:51 UTC 2016 x86_64
- PHP Version: 7.0.12
- Git version: 1.9.1

**Comment by pavelevap** [comment]
Hi @Elethier,

how did you exactly switched your site to HTTPS? Did you use any plugin or make "search and replace" through the database?

Did you try latest [4.0-beta](https://github.com/versionpress/versionpress/releases) version?

Related issue: https://github.com/versionpress/versionpress/issues/1173

**Comment by Elethier** [comment]
I managed to fix the issue! I had to go to `wp-config.php` and change the variables `WP_SITEURL` and `WP_HOME` to use HTTPS instead of HTTP. I'll close this issue.

@pavelevap to answer your question: I'm using a server running Apache, so I made some changes to one of my Apache configuration files to implement the redirect. I didn't have to do any sort of find and replace in the database. The end result is that any requests on an HTTP URL receive a 302 response instead of a normal response. Since VersionPress wasn't built to handle a 302, the console wasn't working.
